lib/promscrape/discovery/ovhcloud/dedicated_server.go

```diff
@@ -0,0 +1,151 @@
+package ovhcloud
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/netip"
+	"net/url"
+	"path"
+	"strconv"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/promutils"
+)
+
+// dedicatedServer struct from API.
+// IP addresses are fetched independently.
+type dedicatedServer struct {
```

Comment:
Would be great to have link to the OVH docs with response fields

Reply:
Docs are already presented in `getVPSDetails` and `getDedicatedServerDetails`. Since multiple APIs are involved, I believe it would be beneficial to provide links to the most important one, as well as the `getXxxDetails` methods, so developers can easily access and review.